README.md

```diff
@@ -1,14 +1,7 @@
-# Garden RTE Resources [![npm version][npm version badge]][npm version link] [![Build Status][build status badge]][build status link] [![Dependency Status][dependency status badge]][dependency status link]<!-- markdownlint-disable -->
+# Garden RTE Resources [![npm version](https://flat.badgen.net/npm/v/@zendeskgarden/ckeditor)](https://www.npmjs.com/package/@zendeskgarden/ckeditor) [![Build Status](https://flat.badgen.net/circleci/github/zendeskgarden/ckeditor/main?label=build)](https://circleci.com/gh/zendeskgarden/ckeditor/tree/main) [![Dependency Status](https://flat.badgen.net/david/dev/zendeskgarden/ckeditor?label=dependencies)](https://david-dm.org/zendeskgarden/ckeditor?type=dev)<!-- markdownlint-disable -->
```

Comment:
Storybook mdx process is unable to handle these reference links. @jzempel is ok to use inline links to allow us to share this documentation with storybook?

Reply:
Another option would be to keep the URLs simple and workaround storybook's bug like we did here: https://github.com/zendeskgarden/svg-icons/blob/main/src/.stories/README.stories.mdx 🤷

Reply:
That is slick. Updated to use this logic.